test/blas_test.hpp

```diff
@@ -204,6 +211,13 @@ class BLAS_Test<blas_test_args<ScalarT_, MetadataT_, ExecutorType_>>
   }
 };
 
+template<typename scalar_t>
+cl::sycl::queue make_queue() {
```

Comment:
make queue should not take type. Why do you need to pass type to make queue?

Reply:
We need to create a BLAS_Test instance of the appropriate type. Unless we don't want to use the BLAS_Test thing anymore? Which is probably the correct move here.